demo/index.html

```diff
@@ -27,105 +30,136 @@
     <div class="c-tab c-tab--block c-tab--main">
       <ul class="c-tab__list c-tab__list">
         <li class="c-tab__list__item" tabindex="0">
-          <a href="//zendeskgarden.github.io/" tabindex="-1">Home</a>
+          <a href="/" tabindex="-1">Home</a>
         </li>
         <li class="c-tab__list__item" tabindex="0">
-          <a href="//zendeskgarden.github.io/assets/" tabindex="-1">Assets</a>
+          <a href="/assets" tabindex="-1">Assets</a>
         </li>
         <li class="c-tab__list__item" tabindex="0">
-          <a href="//zendeskgarden.github.io/css-components/" tabindex="-1">CSS Components</a>
+          <a href="//zendeskgarden.github.io/css-components" tabindex="-1">CSS Components</a>
         </li>
         <li class="c-tab__list__item is-selected" tabindex="0">
-          <a href="/react-components" tabindex="-1">React Components</a>
+          <a href="//zendeskgarden.github.io/react-components/" tabindex="-1">React Components</a>
```

Comment:
Don't change any of the URLs in this section. They are intended to work for both garden.zendesk.com and localhost.

Reply:
That makes sense. Brought the other in from the copy-pasta.